config/routes.rb

```diff
@@ -1,4 +1,5 @@
 Rails.application.routes.draw do
+  get '/customer-service', to: 'customer_service#index', as: 'customer_service'
```

Comment:
So Rails will match routes from the top down in `routes.rb`, halting lookups once it finds a match. I don't imagine we'll have much in the way of collisions, so for organization's sake, would it make sense to move this down above line 5?

```rb
resources :faqs, only: :index
resources :tutorials, only: :index

get '/customer-service', to: 'customer_service#index', as: 'customer_service'
get '/service-specific-information', to: 'service_specific_information#index', as: 'service_specific_information'
```